app/controllers/register/trustees/individual/TrusteeUTRController.scala

```diff
@@ -38,15 +40,40 @@ class TrusteeUTRController @Inject()(val appConfig: FrontendAppConfig,
                                      authenticate: AuthAction,
                                      getData: DataRetrievalAction,
                                      allowAccess: AllowAccessActionProvider,
-                                     requireData: DataRequiredAction
-                                     )(implicit val ec: ExecutionContext) extends FrontendController with Retrievals with I18nSupport with Enumerable.Implicits {
+                                     requireData: DataRequiredAction,
+                                     formProvider: UTRFormProvider
+                                     )(implicit val ec: ExecutionContext) extends UTRController {
+
+  private def form: Form[String] = formProvider()
+
+  private def viewModel(mode: Mode, index: Index, srn: Option[String], trusteeName: String): UTRViewModel = {
+    UTRViewModel(
+      postCall = routes.TrusteeUTRController.onSubmit(mode, index, srn),
+      title = Message("messages__trusteeUtr__title"),
+      heading = Message("messages__trusteeUtr__h1", trusteeName),
+      hint = Message("messages_utr__hint"),
+      subHeading = None,
+      srn = srn
+    )
+  }
+
+  private def trusteeName(index: Index) = Retrieval{ implicit request =>
+    TrusteeNameId(index).retrieve.right.map(_.fullName)
+  }
```

Comment:
Does this need to be in a separate function? It would be neater to just do the retrieve and map the the result into the get